query-node/mappings/src/content/nft.ts

```diff
@@ -345,7 +345,7 @@ export async function createNft(
 
   // calculate some values
   const creatorRoyalty = nftIssuanceParameters.royalty.isSome
-    ? nftIssuanceParameters.royalty.unwrap().toNumber()
+    ? nftIssuanceParameters.royalty.unwrap().div(new BN(10000000)).toNumber()
```

Comment:
Let's properly define and name this magic constant; put it into `query-node/mappings/src/content/utils.ts` and import it into test `Api.ts` instead of the same magic constant that's used there. https://github.com/Joystream/joystream/blob/ba96a82dab2bff6533c371dfd1810d92b8aeecac/tests/network-tests/src/Api.ts#L823

Also, the number can be written in a little bit easier to read format `new BN(10_000_000)`.